packages/core/core/src/TargetDescriptor.schema.js

```diff
@@ -112,6 +115,9 @@ export const DESCRIPTOR_SCHEMA: SchemaEntity = {
     distEntry: {
       type: 'string',
     },
+    source: {
+      type: 'string',
+    },
```

Comment:
This shouldn't be needed because you already added it to `PACKAGE_DESCRIPTOR_SCHEMA.properties `